anta/tests/cvx.py

```diff
@@ -87,3 +87,47 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyActiveCVXConnections(AntaTest):
+    """Verifies the expected number of active CVX Connections.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the expected number of connections is present.
+    * Failure: The test will fail if the unexpected number of connections is present.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+      - VerifyActiveCVXConnections:
+          expected_connection_count: 100
+    ```
+    """
+
+    categories: ClassVar[list[str]] = ["cvx"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show cvx connections brief", revision=1)]
```

Comment:
1/ we will need to handle this return somewhere in ANTA let's maybe add it in a TODO for now

```
site3-wan1#show cvx connections brief
% Unavailable command (controller not ready)
```

2/ are you sure the revision is the correct one?

Reply:
```suggestion
    # TODO: @gmuloc - cover "% Unavailable command (controller not ready)"
    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show cvx connections brief", revision=1)]
```